src/Aqovia.Utilities.SagaMachine/StatePersistance/RedisKeyValueStore.cs

```diff
@@ -8,7 +10,7 @@ namespace Aqovia.Utilities.SagaMachine.StatePersistance
 {
     public class RedisKeyValueStore : IKeyValueStore, IDisposable
     {
-        private const double DefaultLockExpiryTime = 500;
+        private const double DefaultLockExpiryTime = 30000;
```

Comment:
@taliesins the [documentation comment](https://github.com/Aqovia/Aqovia.SagaMachine/pull/5/files#diff-ae70b8b9ab807d54df6dc14e5f1de720R115) still says 500 milliseconds.

While we're at it, @RavindraSolanki , what about that suggestion I had to remove this method from the interface, and rather implement it as an extension method on `IKeyValueStore`, so that implementors of the underlying data stores don't need to be preoccupied with what that value should be?